FeedStoreChallenge/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,27 @@
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be `final`
Please make the class `final`.

```suggestion
final class ManagedFeedImage: NSManagedObject {
```